hledger/test/balance/balance.test

```diff
@@ -217,3 +206,19 @@ $ hledger -f- bal --count -1
                    1  b
 --------------------
                    3  
+
+# ** 16. balance --flat --empty does not display accounts which have not been
+# seen, even if they're implied, but does show accounts that have been seen
+# with 0 balance.
```

Comment:
I'm not sure what this means in userese ?

Reply:
I've changed the test description. Let me know if it's clearer.